src/Symfony/Component/JsonEncoder/Attribute/MaxDepth.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\Attribute;
+
+/**
+ * Defines the maximum encoding depth for the property.
+ *
+ * When the maximum depth is reached, a {@see Symfony\Component\JsonEncoder\Encode\Normalizer\NormalizerInterface} service is called if its id has been defined.
```

Comment:
> is called if its id has been defined.

What does that mean? Does one needs to do something specific for such id to be "defined"? The sentence is a bit abstract